compute_endpoint/globus_compute_endpoint/endpoint/config/config.py

```diff
@@ -94,6 +94,12 @@ class Config(RepresentationMixin):
         Designates the endpoint as a multi-user endpoint
         Default: None
 
+    public : bool | None
+        Indicates if all users can discover the multi-user endpoint. Please note that
+        this field does not control access to the endpoint, so it should not be used
+        as a security feature.
+        Default: None
+
```

Comment:
- I might more explicitly call out that this is only relevant for multi-user endpoints, and is forced to false for non-multi-user endpoints.  (Forced by way of 422, but still.)
- Might mention what "discover" means.  It's only `config` text, but giving enough of a hint will be appreciated by those reading this who aren't devs.
- This documentation begs a fleshing out of the `multi_user` flag as well, discussing that multi-user endpoints are _always_ accessible; administrators must use the identity mapper to delegate access.

Reply:
I updated the wording in a fresh commit. Let me know what you think.